i18n/spec/utilSpec.js

```diff
@@ -1,16 +0,0 @@
-'use strict';
-
-var util = require('../src/util.js');
-
-describe('findLocaleId', function() {
-  it('should find localeId', function() {
-    expect(util.findLocaleId('', 'num')).toBeUndefined();
-    expect(util.findLocaleId('aa', 'datetime')).toBeUndefined();
-    expect(util.findLocaleId('aa', 'randomType')).toBeUndefined();
-    expect(util.findLocaleId('NumberFormatSymbols_en', 'datetime')).toBeUndefined();
-    expect(util.findLocaleId('DateTimeSymbols_en', 'num')).toBeUndefined();
-
-    expect(util.findLocaleId('DateTimeSymbols_en', 'datetime')).toBe('en');
-    expect(util.findLocaleId('NumberFormatSymbols_en_US', 'num')).toBe('en_US');
-  });
-});
```

Comment:
`closureI18nExtractor` implements (and exports) a `findLocaleId` (which is just a slightly enhanced version of `util.findLocaleId`. There are some tests for `findLocaleId` in `closureI18nExtractorSpec`, but `util` had some more tests.
Anyway, this is minor - we didn't have the tests before and they are not related to this PR, so it's fine to leave things as they are now (we have piggybacked enough on this PR :stuck_out_tongue:)